node/lib/util/status.js

```diff
@@ -204,8 +204,8 @@ from ${colors.red(GitUtil.shortSha(status.commitSha))}.
                     case RELATION.UNKNOWN:
                         result += `
 Change staged to commit ${colors.yellow(GitUtil.shortSha(status.indexSha))} \
-but cannot verify relation to \
-${colors.yellow(GitUtil.shortSha(status.commitSha))} as the repo is closed.
+but cannot verify relation to previous commit \
```

Comment:
This message does not seem actionable to me.  If I saw it, I would not be sure what to do to fix it, nor what the implications of not fixing it would be .